apps/vanilla/src/components/animeCard.ts

```diff
@@ -0,0 +1,75 @@
+import { Anime } from '@js-camp/core/models/anime';
+import { AnimeStatus } from '@js-camp/core/enums/statusType';
+
+import { getElement } from '../utils/utils';
+
+/** AnimeCard. */
+export class AnimeCard {
+  /** Id. */
+  public readonly id: number;
+
+  /** Image URL. */
+  public readonly imageURL: string;
+
+  /** TitleEng. */
+  public readonly titleEng: string;
+
+  /** TitleJpn. */
+  public readonly titleJpn: string;
+
+  /** Type. */
+  public readonly type: string;
+
+  /** Status. */
+  public readonly status: AnimeStatus;
+
+  /** AiredStart. */
+  public readonly airedStart: Date;
+
+  public constructor(anime: Anime) {
+    this.id = anime.id;
+    this.titleEng = anime.titleEng;
+    this.titleJpn = anime.titleJpn;
+    this.imageURL = anime.image;
+    this.airedStart = anime.airedStart;
+    this.type = anime.type;
+    this.status = anime.status;
+  }
+
+  /** Create and initiate an anime card from a template. Return HTML element.
+   * @param templateSelector Template selector.
+   */
```

Comment:
Please, format **all** multiline jsdoc comments properly.
```suggestion
  /**
   * Create and initiate an anime card from a template. Return HTML element.
   * @param templateSelector Template selector.
   */
```

Reply:
Done